mod/metrics/set/type_plus_right/metric_value/checked_by.rb

```diff
@@ -1,46 +1,82 @@
 def user
-  Card.fetch(Auth.current_id)
+  Auth.current
 end
 
-def user_checked_before
-  return true if checked_users.include?user.name
+def user_checked?
+  !check_requested? && checkers.include?(user.name)
 end
 
-def checked_users
+def checkers
   item_names
 end
 
+def checked?
+  !check_requested? && checkers.present?
+end
+
+def check_requested?
+  item_names.first == "request"
+end
+
+def check_requester
+  item_names.last
+end
+
+def option_names
+  ["request"]
+end
+
 format :html do
   # view :new do
   #   return _render_double_check_view
   # end
   view :missing do |args|
     if card.new_card? && card.left
-      Auth.as_bot do
-        card.save!
-      end
-      render(@denied_view, args)
+      Auth.as_bot { card.save! }
+      render @denied_view, args
```

Comment:
It would be great if you could try to get rid of this `save!` call.  I know it's not originally your mess, but it's still adding junk data.